marlowe-cli/examples/cfd/run-cfd.sh

```diff
@@ -778,43 +778,45 @@ cardano-cli query utxo --testnet-magic "$MAGIC" --address "$ORACLE_ADDRESS" | se
-                               --tx-in "$TX_7"#1                                        \
-                               --tx-in "$TX_7"#2                                        \
-                               --tx-out "$BURN_ADDRESS+1400000+1 $COUNTERPARTY_TOKEN" \
-                               --required-signer "$COUNTERPARTY_PAYMENT_SKEY"           \
-                               --change-address "$FAUCET_ADDRESS"                       \
-                               --out-file /dev/null                                     \
-                               --submit 600
 
-cardano-cli query utxo --testnet-magic "$MAGIC" --address "$COUNTERPARTY_ADDRESS"
+echo "Sending back funds:"
 
-marlowe-cli transaction simple --testnet-magic "$MAGIC"                         \
-                               --socket-path "$CARDANO_NODE_SOCKET_PATH"                \
-                               --tx-in "$TX_8"#0                                        \
-                               --tx-in "$TX_8"#1                                        \
-                               --tx-in "$TX_8"#2                                        \
-                               --tx-out "$BURN_ADDRESS+1400000+1 $ORACLE_TOKEN" \
-                               --required-signer "$ORACLE_PAYMENT_SKEY"           \
-                               --change-address "$FAUCET_ADDRESS"                       \
-                               --out-file /dev/null                                     \
-                               --submit 600
+marlowe-cli -- util faucet --testnet-magic "$MAGIC" \
+                           --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                           --all-money \
+                           --faucet-address "$ORACLE_ADDRESS"        \
+                           --required-signer "$ORACLE_PAYMENT_SKEY"     \
+                           --submit 600 \
+                           --out-file /dev/null \
+                           "$FAUCET_ADDRESS"
+
+marlowe-cli -- util faucet --testnet-magic "$MAGIC" \
+                           --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                           --all-money \
+                           --faucet-address "$PARTY_ADDRESS" \
+                           --required-signer "$PARTY_PAYMENT_SKEY" \
+                           --submit 600 \
+                           --out-file /dev/null \
+                           "$FAUCET_ADDRESS"
+
+marlowe-cli -- util faucet --testnet-magic "$MAGIC" \
+                           --socket-path "$CARDANO_NODE_SOCKET_PATH" \
+                           --all-money \
+                           --faucet-address "$COUNTERPARTY_ADDRESS" \
+                           --required-signer "$COUNTERPARTY_PAYMENT_SKEY" \
+                           --submit 600 \
+                           --out-file /dev/null \
+                           "$FAUCET_ADDRESS"
 
-cardano-cli query utxo --testnet-magic "$MAGIC" --address "$ORACLE_ADDRESS"
```

Comment:
I think that those `cardano-cli query utxo..` calls were pretty useful because they showed the final results of the cleanup. Maybe we can bring them back?

Reply:
Thanks @paluh - I totally missed this!